config/macsec.py

```diff
@@ -0,0 +1,168 @@
+#
+@macsec_port.command('add')
+@click.argument('port', metavar='<port_name>', required=True)
+@click.argument('profile', metavar='<profile_name>', required=True)
+@clicommon.pass_db
+def add_port(db, port, profile):
+    """
+    Add MACsec port 
+    """
+    ctx = click.get_current_context()
+
+    if clicommon.get_interface_naming_mode() == "alias":
+        alias = port
+        iface_alias_converter = clicommon.InterfaceAliasConverter(db)
+        port = iface_alias_converter.alias_to_name(alias)
+        if port is None:
+            ctx.fail("cannot find port name for alias {}".format(alias))
+
+    profile_entry = db.cfgdb.get_entry('MACSEC_PROFILE', profile)
+    if len(profile_entry) == 0:
+        ctx.fail("profile {} doesn't exist".format(profile))
+
+    db.cfgdb.set_entry("PORT", port, {'macsec': profile})
+
+
+#
+# 'del' command ('config port del ...')
+#
+@macsec_port.command('del')
+@click.argument('port', metavar='<port_name>', required=True)
+@clicommon.pass_db
+def del_port(db, port):
+    """
+    Delete MACsec port
+    """
+    ctx = click.get_current_context()
+
+    if clicommon.get_interface_naming_mode() == "alias":
+        alias = port
+        iface_alias_converter = clicommon.InterfaceAliasConverter(db)
+        port = iface_alias_converter.alias_to_name(alias)
+        if port is None:
+            ctx.fail("cannot find port name for alias {}".format(alias))
+
+    db.cfgdb.set_entry("PORT", port, {'macsec': ""})
+
+
+#
+# 'profile' group ('config macsec profile ...')
+#
+@macsec.group(cls=clicommon.AbbreviationGroup, name='profile')
+def macsec_profile():
+    pass
+
+
+def is_hexstring(hexstring: str):
+    try:
+        int(hexstring, 16)
+        return True
+    except ValueError:
+        return False
+
+
+#
+# 'add' command ('config macsec profile add ...')
+#
+@macsec_profile.command('add')
+@click.argument('profile', metavar='<profile_name>', required=True)
+@click.option('--priority', metavar='<priority>', required=False, default=255, show_default=True, type=click.IntRange(0, 255), help="For Key server election. In 0-255 range with 0 being the highest priority.")
+@click.option('--cipher_suite', metavar='<cipher_suite>', required=False, default="GCM-AES-128", show_default=True, type=click.Choice(["GCM-AES-128", "GCM-AES-256", "GCM-AES-XPN-128", "GCM-AES-XPN-256"]), help="The cipher suite for MACsec.")
+@click.option('--primary_cak', metavar='<primary_cak>', required=True, type=str, help="Primary Connectivity Association Key.")
+@click.option('--primary_ckn', metavar='<primary_cak>', required=True, type=str, help="Primary CAK Name.")
+@click.option('--policy', metavar='<policy>', required=False, default="security", show_default=True, type=click.Choice(["integrity_only", "security"]), help="MACsec policy. INTEGRITY_ONLY: All traffics, except EAPOL, will be converted to MACsec packets without encryption.SECURITY: All traffics, except EAPOL, will be encrypted by SecY.")
```

Comment:
Minor typos:  change traffics to traffic and add space between period and SECURITY.